spec/unit_spec_helper.rb

```diff
@@ -31,8 +23,17 @@
   UnitTests::ActiveModelVersions.configure_example_group(config)
   UnitTests::DatabaseHelpers.configure_example_group(config)
   UnitTests::ColumnTypeHelpers.configure_example_group(config)
+  UnitTests::ValidationMatcherScenarioHelpers.configure_example_group(config)
 
   config.include UnitTests::Matchers
+
+  config.infer_spec_type_from_file_location!
+  config.example_status_persistence_file_path = "spec/examples.txt"
+  config.alias_it_behaves_like_to(:it_supports, "it supports")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.